notebooks/text_models/solutions/rnn_encoder_decoder.ipynb

```diff
@@ -4,25 +4,33 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    decoder_model = keras.Model(
</pre><p><code>Model</code> seems to be imported already.</p><p> </p><p> <code>ecoder_model = Model(...)</code>  should be fine as the training model. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/719/discussion/#comment-2431166184'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>